doc/features/mapper/limitations-and-faq/README.md

```diff
@@ -0,0 +1,56 @@
+# Limitations and Frequently Asked Questions
+
+## Limitations
+
+There are some limitations in the Mapper design:
+
+- Secondary indexes and SOLR queries are not supported.
+- Case sensitive CQL identifiers (enclosed with double quotation marks) are not supported.
```

Comment:
Ah, i just realized that it would be problematic to deal with as column names get assigned to fields, but maybe there could be a way to make it work if the users provide the mappings explicitly.

Reply:
I've filed https://datastax-oss.atlassian.net/browse/NODEJS-466